lib/renderer.js

```diff
@@ -1116,6 +1116,26 @@
         
         return component.document.layers.visit(layerHasEffects);
     };
+
+    /**
+     * Check if the document returns empty bounds.
+     * 
+     * @private
+     * @param {Document} document incoming document where empty layers will be searched for.
+     * @return {boolean}
+     */
+    PixmapRenderer.prototype._documentHasZeroBounds = function (document) {
+
+        var docBounds = this._getContentBounds(document.layers, true);
+        if (docBounds.top === 0 &&
+            docBounds.left === 0 &&
+            docBounds.bottom === 0 &&
+            docBounds.right === 0) {
```

Comment:
Is it enough to check for this specific case, or should it check for more general case where "area is empty" (which also catches first case)? For example:

```js
        if ((docBounds.bottom - docBounds.top) <= 0 ||
            (docBounds.right - docBounds.left) <= 0) {
``

If so, might want to rename function.